src/config/config.ts

```diff
@@ -44,13 +44,22 @@ export class Configuration {
     password: process.env.MYDEFICHAIN_PASSWORD,
   };
 
+  kyc = {
+    kycSecret: process.env.KYC_SECRET,
+    allowedWebhookIps: process.env.KYC_WEBHOOK_IPS?.split(','),
+    apiUrl: process.env.KYC_API_URL,
+  };
+
   auth = {
     jwt: {
       secret: process.env.JWT_SECRET,
       signOptions: {
         expiresIn: process.env.JWT_EXPIRES_IN ?? 172800,
       },
     },
+    lockSeed: process.env.LOCK_SEED?.split(','),
+    signMessage:
+      'By_signing_this_message,_you_confirm_that_you_are_the_sole_owner_of_the_provided_DeFiChain_address_and_are_in_possession_of_its_private_key._Your_ID:_',
```

Comment:
Use a "blockchain-netural" message: `By_signing_this_message,_you_confirm_that_you_are_the_sole_owner_of_the_provided_Blockchain_address._Your_ID:_` and perhaps add Lock, so it's different from DFX?